lms/urls.py

```diff
@@ -20,9 +20,7 @@
 from openedx.features.enterprise_support.api import enterprise_enabled
 
 
-# Uncomment the next two lines to enable the admin:
-if settings.DEBUG or settings.FEATURES.get('ENABLE_DJANGO_ADMIN_SITE'):
-    admin.autodiscover()
+django_autodiscover()
```

Comment:
Nit: I'd prefer that cms/urls.py and lms/urls.py did autodiscover the exact same way.